src/uct/ib/mlx5/ib_mlx5_hw.c

```diff
@@ -173,6 +173,12 @@ ucs_status_t uct_ib_mlx5dv_init_obj(uct_ib_mlx5dv_t *obj, uint64_t obj_type)
                 ucs_container_of(obj->dv.srq.out, uct_ib_mlx5dv_srq_t, dv));
     }
 
+#if HAVE_IBV_EXP_DM
```

Comment:
should it be HAVE_IBV_DM? since it uses MLX5DV_OBJ_DM?

Reply:
It's fallback when there are no mlx5dv

Reply:
so why need `#if`? maybe it should be inside uct_ib_mlx5_get_dm_info?

Reply:
if not defined HAVE_IBV_EXP_DM neither is MLX5DV_OBJ_DM - won't compile

Reply:
ok